src/core/LanguageMenu/LanguageMenu.tsx

```diff
@@ -84,6 +84,7 @@ const LanguageMenuPopoverPosition = (
     left: `${targetRect.left - popoverRect.width + targetRect.width}px`,
     // eslint-disable-next-line no-undef
     top: `${targetRect.top + targetRect.height + window.pageYOffset}px`,
+    maxWidth: `${targetRect.width + targetRect.left - 20}px`,
```

Comment:
Currently using **20px** as a gap and this seemed to be ok for PAMU also.

But should we use token or somewhat create local variable that gives the explanation, like:
```typescript
const gap = 20
//...
maxWidth: `${targetRect.width + targetRect.left - gap}px`,
```

Reply:
As proposed via chat, we could also use that min. of 20px margin or targetRect width, which ever is bigger at the moment. This way the min size would be the size of the targetRect (-2px) and in would allow the popover to grow to max 20px from left side.

Reply:
_As discussed further, this gap is now 30px instead._